iface/interface.go

```diff
@@ -7,11 +7,15 @@ import (
 	"strings"
 )
 
+type File struct {
+	Package    string
+	Imports    []Import
```

Comment:
I'm sure there's some magic I'm missing here - but what happens if I import a module in a file, but don't use that module in any of the mocked interfaces in that file? does it just get included? or has this circumstance been so rare I've never noticed it?

Reply:
Nevermind I see the imports cleanup step